src/components/settings/LocationSettings.js

```diff
@@ -53,8 +53,7 @@ export const LocationSettings = () => {
       Location.getForegroundPermissionsAsync().then((response) => {
         // if the system permission is granted, we can simply enable the sorting
         if (response.status === Location.PermissionStatus.GRANTED) {
-          const newSettings = { locationService: true };
-          setAndSyncLocationSettings(newSettings);
+          setAndSyncLocationSettings({ locationService: true });
```

Comment:
I'm curious what is the benefit of setting locationService to true here instead of getting it from newSettings?

Reply:
reducing the variables. it was only used for the next line without any benefit.